libraries/core/src/main/kotlin/com/bumble/appyx/core/node/Node.kt

```diff
@@ -57,7 +57,7 @@ open class Node(
     val isRoot: Boolean =
         ancestryInfo == AncestryInfo.Root
 
-    private val parent: ParentNode<*>? =
+    protected val parent: ParentNode<*>? =
```

Comment:
In RIBs it is a public property. Wanna align?

Reply:
let's align